App.js

```diff
@@ -1,26 +1,47 @@
-import React from 'react'
-import styled from 'styled-components/native'
+import React, { useState } from 'react';
+import { StyleSheet, View, ScrollView, ImageBackground, Button } from 'react-native';
+import styled from 'styled-components/native';
+import Weather from './components/Weather';
+import Compass from './components/Compass';
+import Heading from './components/Heading';
+import MyLocation from './components/MyLocation';
```

Comment:
I get some errors when cloning and running your app locally because the components folder is named with capital C - Components. I had to change that folder name to make it work. Take a look at this as I know github is a bit messy when it comes to renaming folders like that.